packages/databases-collections-list/src/index.spec.tsx

```diff
@@ -121,5 +140,27 @@ describe('databases and collections list', function () {
 
       expect(connectionClickSpy).to.be.called;
     });
+
+    it('should not display statistics (except storage size) on timeseries collection card', function () {
+      render(
+        <CollectionsList
+          connectionTitle="My Connection"
+          databaseName="My Database"
+          collections={colls}
+          onClickConnectionBreadcrumb={() => {}}
+          onCollectionClick={() => {}}
+        ></CollectionsList>
+      );
+
+      const timeseriesCard = screen
+        .getByText('bat.bat')
+        .closest('[data-testid="collection-grid-item"]');
+      expect(timeseriesCard).to.exist;
+      expect(timeseriesCard).to.contain.text('Storage size:');
+      expect(timeseriesCard).to.not.contain.text('Documents:');
+      expect(timeseriesCard).to.not.contain.text('Avg. document size::');
+      expect(timeseriesCard).to.not.contain.text('Indexes:');
+      expect(timeseriesCard).to.not.contain.text('Total index size:');
```

Comment:
I think there's a chance that this PR ended up including a few commits you did not intend to include? (Not a problem, this can be fixed through git, but feel free to ask on Slack if you're unsure how to do that :))